src/gobprepare/prepare_client.py

```diff
@@ -254,6 +255,59 @@ def action_check_row_counts(self, action: RowCountConfig) -> bool:
                 )
         return tables_counts_ok
 
+    def action_check_source_sync_complete(self, action: SyncSchemaConfig) -> bool:
+        """Check source sync action. Verify when sync tables from databricks is completed.
+
+        :param action:
+        :return:
+        """
+        table_name = action["table_name"]
+        schema = action["schema"]
+
+        if not self._dst_datastore.query(check_table_existence_query("public", table_name)):  # type: ignore[union-attr]
+            raise GOBException(f"Table 'public.{table_name}' does not exists.")
+
+        self._check_last_schema_sync(table_name, schema)
+        update_columns = ["last_prepare_start = CURRENT_TIMESTAMP", "last_prepare_end = NULL"]
+        self._update_sync_schema(table_name, schema, update_columns)
+        logger.info(f"Prepare for '{schema}' started.")
+
+        return True
```

Comment:
Bit meaningless. We either return True or raise an exception.

Reply:
Ga ik morgen met jou over hebben!

Reply:
Verwerkt